PyRIGS/settings.py

```diff
@@ -45,8 +45,10 @@
 INTERNAL_IPS = ['127.0.0.1']
 
 ADMINS = (
-    ('Tom Price', 'tomtom5152@gmail.com')
+    [('Tom Price', 'tomtom5152@gmail.com'), ('IT Manager', 'it@nottinghamtec.co.uk'), ('Arona Jones', 'arona.jones@nottinghamtec.co.uk')]
```

Comment:
Why is this wrapped in parenthesis?

Reply:
This is correct, you need to provide a `list of sets`, with each set consisting of the pair `(<display name>, <email>)`. The list is there so it can iterate over the sets to send the email to multiple destinations.